### DISCUSSION TITLE
Rpubs Error: Can't RE-publish documents

### DISCUSSION THREAD

**Discussion by naturallyfunny** [teamalgoritma/community#798] [Q&A] [answered]
## Error:
```
Error in curl::curl_fetch_memory(url, handle = handle) : 
  HTTP/2 stream 1 was reset
Calls: <Anonymous> -> http -> <Anonymous>
Execution halted
```
<img width="628" alt="Screenshot 2023-11-18 at 5 18 18 PM" src="https://github.com/teamalgoritma/community/assets/134937547/ace25d7c-f9bf-4656-aefc-17f5b91ada87">

## Device:
Macbook Air M1 2020

## Pertanyaan:
Saya mendapatkan error tersebut setiap kali ingin republish document. Ada yang tau solusinya?

**Comment by IrfanChairurrachman** [comment, marked as answer]
Dear @ArdiantheGreat ,

Permasalahan tidak bisa republish rpubs akhir-akhir ini terkadang dialami oleh sebagian orang. Terdapat beberapa alternatif solusi yang mungkin dapat membantu, antara lain:

1. Mencoba republish di waktu yang berbeda, terkadang permasalahan ini dikarenakan dari server rpubs itu sendiri.
2. Membuat rmd baru baik di folder yang sama ataupun di folder lain kemudian melakukan publish ulang (rpubs yang lama artikelnya dihapus)
3. Jika 2 cara di atas masih tidak solve, Anda dapat mencoba downgrading rsconnect Anda ke versi "0.8.29" dengan cara berikut:
    ```
    remove.packages("rsconnect") #Remove Installed Packages
   remotes::install_version("rsconnect", version = "0.8.29") #Installing a Specific Version of a Package
    ```
    Setelah itu silahkan upload rmd baru ke rpubs dan seharusnya dengan rmd baru ini Anda bisa melakukan republish lagi. Link [forum](https://community.rstudio.com/t/republishing-to-rpubs-error-in-curl-curl-fetch-memory-url-handle-handle/171386)

Sekian jawaban dari kami. Bila ternyata informasi ini dapat menjawab pertanyaan Anda, Anda dapat “Mark as answer” jawaban ini.

**Reply by IrfanChairurrachman** [reply]
Halo @ArdiantheGreat , untuk menjaga agar forum diskusi berjalan kondusif, kami izin menandai pertanyaan "mark as answer" dan melakukan close pada diskusi ini karena belum ada respon setelah 7 hari respon terakhir. Semoga dapat dipahami. Terima kasih.
